### PR TITLE
feat: add plans to deploy eks cluster for ecom worker process

### PR DESCRIPTION
**Pull request by gabeobrien**

### CODE REVIEW

File: plans/k8s/eks-worker.yaml [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+    aws-k8s-worker:
```
Comment (ipcrm) [inline plans/k8s/eks-worker.yaml:2]: need to fix this indent

File: plans/util/aws_eks_audit_log_worker.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+    aws-eks-audit-log-worker:
```
Comment (ipcrm) [inline plans/util/aws_eks_audit_log_worker.yml:2]: fix indent

File: plans/k8s/eks-worker.yaml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+          extension: EKS
+          description: This step uses the EKS extension (Terraform under the hood) to deploy the actual cluster. Expect runtimes in excess of 10 minutes
+          args:
+            region: !secret aws.region
+            cluster_name: workereks
+          helpers:
+            - helper: EKSKubectl
+              lifecycle: [ afterCreate ]
+
+        - name: lacework-agent-legacy
```
Comment (ipcrm) [inline plans/k8s/eks-worker.yaml:29]: probably no need for this entire block; or the conditions here or in the lacework-agent step

this was only left in the original k8s plan for compatibility

**Review by ipcrm — CHANGES_REQUESTED** [review]
A few minor notes; otherwise looks great!

**Comment by gabeobrien** [comment]
@ipcrm fixed!

**Review by ipcrm — APPROVED** [review]
:shipit: !!!

**Pull request merged**